### DISCUSSION TITLE
Unknown sources feature?

### DISCUSSION THREAD

**Discussion by Brakfor** [alvr-org/ALVR#368] [Q&A] [answered]
Like can you make playing games not only in steam?

**Comment by zmerp** [comment, marked as answer]
The SteamVR runtime is a requirement, but you can play Oculus games with Revive.

**Reply by Brakfor** [reply]
where i can download steamvr runtime?

**Reply by zmerp** [reply]
Sorry, I was not clear. Just install SteamVR and Revive
